src/ResourceManager/Sql/Commands.Sql/ServerKeyVaultKey/Services/AzureSqlServerKeyVaultKeyAdapter.cs

```diff
@@ -84,13 +84,10 @@ public IList<AzureSqlServerKeyVaultKeyModel> List(string resourceGroupName, stri
         /// <returns>The updated server key Vault Key model</returns>
         public AzureSqlServerKeyVaultKeyModel CreateOrUpdate(AzureSqlServerKeyVaultKeyModel model)
         {
-            var resp = Communicator.CreateOrUpdate(model.ResourceGroupName, model.ServerName, model.ServerKeyName, new ServerKeyCreateOrUpdateParameters()
+            var resp = Communicator.CreateOrUpdate(model.ResourceGroupName, model.ServerName, model.ServerKeyName, new ServerKey()
```

Comment:
I removed any changes relating to updating the ServerKey and EncryptionProtector related cmdlets to the Swagger generated SDK, so that this change is only related to Identity changes for Server.